docs/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -0,0 +1,89 @@
+```
+{
+    "forgerStakeInfo": {
+        "ownerAddress": "e8674062598b86d8258257a4d79e99d15594d864", 
+        "blockSignPublicKey": "e8674062598b86d8258257a4d79e99d15594d864", 
+        "vrfPubKey": "4439cbfd50af1b846e5ef06889d3192ef7a459bdd4640dc6da506062de43113c80", 
+        "value": 10000
+    }
+}
+```
+where:
+- `ownerAddress`: is the address that is requesting to stake such amount and the only one allowed to unstake (withdraw) the specified amount;
+- `blockSignPublicKey`: is the address that will sign the block when forged. It can be the same address as the ownerAddress or a different one (delegate forging); if this value is not specified the ownerAddress will be used;
+- `vrfPubKey`: this is the key created in the previous step;
+- `value`: the amount of ZEN expressed in Satoshis (10^8) to put on stake, that will represent your voting power.
+
+#### How to create a forging stake using web3js
+Below there is an example (written in javascript to run in Remix IDE) of how to create the forging stake by interacting with the contract:
+```
+(async () => {
+    try {
+
+        console.log('Running testWeb3 - Delegate or create a new Forging Stake script...')
+
+        const smartContractAddress = "0x0000000000000000000022222222222222222222";
+        const amount = 0.001; // amount of ZEN of the new stake
+        const accounts = await web3.eth.getAccounts();
+        console.log('Account ' + accounts[0]);
+
+        const abi = require("browser/contracts/artifacts/abi_fs.json"); // Change this for different path
+
+        const contract = new web3.eth.Contract(abi, smartContractAddress, {from: accounts[0]});
+
+        // Testing method:  delegate
+        const ownerAddress = accounts[0];
+        // pick one of the existing forgers, or create a new forger
+        const blockSignPublicKey = "0x" + YOUR_BLOCK_SIGN; // the address that will sign the block when forged; it can be the same address as the ownerAddress or a different one (delegate forging); if this value is not specified the ownerAddress will be used
+        const forgerVrfPublicKey = "0x" + YOUR_VRF_KEY; // this is the key created in the previous step
+        const first32BytesForgerVrfPublicKey = forgerVrfPublicKey.substring(0, 66);
+        const lastByteForgerVrfPublicKey = "0x" + forgerVrfPublicKey.substring(66, 68);
+
+        methodName = 'delegate';
+        console.log('Response for '+ methodName);
+
+        const response = await contract.methods.delegate(blockSignPublicKey,first32BytesForgerVrfPublicKey,lastByteForgerVrfPublicKey,ownerAddress).send({value: amount *10**18}).then(console.log);
+
+    } catch (e) {
+        console.log("Error:" + e.message);
+    }
+  })()
+```
+⚠️ **Please, remember to replace YOUR_BLOCK_SIGN and YOUR_VRF_KEY with the actual keys, and make sure to specify the path of the native contract ABI (not provided in the documentation, but available here together with the complete Remix workspace)!**  
```

Comment:
replace with:

**Note:** Remember to replace **YOUR_BLOCK_SIGN** and **YOUR_VRF_KEY** with the actual keys. Make sure to specify the path of the native contract ABI (not provided in the documentation, but available here together with the complete Remix workspace)!